frontend/src/utils/customHooks/useDisableElementsOnSomeRoutes.js

```diff
@@ -0,0 +1,21 @@
+import { useState, useEffect } from 'react'
+import { useLocation } from 'react-router-dom'
+
+export const useDisableElementsOnSomeRoutes = props => {
```

Comment:
It's not a component, so `props` is misleading. `paths` conveys more meaning.

Reply:
Right.